adapters/Mintegral/Mintegral.xcodeproj/project.pbxproj

```diff
@@ -38,6 +38,16 @@
 		23D49EEE2899F455006152D8 /* MintegralAdapter.h in Headers */ = {isa = PBXBuildFile; fileRef = 23D49EED2899F455006152D8 /* MintegralAdapter.h */; };
 		23D49EF1289B1DF6006152D8 /* GADMAdapterMintegralUtils.h in Headers */ = {isa = PBXBuildFile; fileRef = 23D49EEF289B1DF6006152D8 /* GADMAdapterMintegralUtils.h */; };
 		23D49EF2289B1DF6006152D8 /* GADMAdapterMintegralUtils.m in Sources */ = {isa = PBXBuildFile; fileRef = 23D49EF0289B1DF6006152D8 /* GADMAdapterMintegralUtils.m */; };
+		AD18D66628AE36C5001DF8BA /* GADMAdapterMintegralExtras.m in Sources */ = {isa = PBXBuildFile; fileRef = AD18D65C28AE36C5001DF8BA /* GADMAdapterMintegralExtras.m */; };
+		AD18D66728AE36C5001DF8BA /* GADMAdapterMintegralInterstitialRenderer.h in Headers */ = {isa = PBXBuildFile; fileRef = AD18D65E28AE36C5001DF8BA /* GADMAdapterMintegralInterstitialRenderer.h */; };
+		AD18D66828AE36C5001DF8BA /* GADMAdapterMintegralBannerRenderer.h in Headers */ = {isa = PBXBuildFile; fileRef = AD18D65F28AE36C5001DF8BA /* GADMAdapterMintegralBannerRenderer.h */; };
+		AD18D66928AE36C5001DF8BA /* GADMAdapterMintegralRewardedAdRenderer.h in Headers */ = {isa = PBXBuildFile; fileRef = AD18D66028AE36C5001DF8BA /* GADMAdapterMintegralRewardedAdRenderer.h */; };
+		AD18D66A28AE36C5001DF8BA /* GADMAdapterMintegralNativeRenderer.m in Sources */ = {isa = PBXBuildFile; fileRef = AD18D66128AE36C5001DF8BA /* GADMAdapterMintegralNativeRenderer.m */; };
+		AD18D66B28AE36C5001DF8BA /* GADMAdapterMintegralBannerRenderer.m in Sources */ = {isa = PBXBuildFile; fileRef = AD18D66228AE36C5001DF8BA /* GADMAdapterMintegralBannerRenderer.m */; };
+		AD18D66C28AE36C5001DF8BA /* GADMAdapterMintegralInterstitialRenderer.m in Sources */ = {isa = PBXBuildFile; fileRef = AD18D66328AE36C5001DF8BA /* GADMAdapterMintegralInterstitialRenderer.m */; };
+		AD18D66D28AE36C5001DF8BA /* GADMAdapterMintegralRewardedAdRenderer.m in Sources */ = {isa = PBXBuildFile; fileRef = AD18D66428AE36C5001DF8BA /* GADMAdapterMintegralRewardedAdRenderer.m */; };
+		AD18D66E28AE36C5001DF8BA /* GADMAdapterMintegralNativeRenderer.h in Headers */ = {isa = PBXBuildFile; fileRef = AD18D66528AE36C5001DF8BA /* GADMAdapterMintegralNativeRenderer.h */; };
+		AD18D67028AE36E3001DF8BA /* GADMAdapterMintegralExtras.h in Headers */ = {isa = PBXBuildFile; fileRef = AD18D66F28AE36E3001DF8BA /* GADMAdapterMintegralExtras.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
Can you remove the settings here as well?

Reply:
I didn't understand what you meant, this was automatically generated when I added the file.